core/type.go

```diff
@@ -0,0 +1,33 @@
+package core
+
+type OidcConfigResponse struct {
+	AuthorizationEndpoint string `json:"authorization_endpoint"`
+	TokenEndpoint         string `json:"token_endpoint"`
+	EndSessionEndpoint    string `json:"end_session_endpoint"`
+	RevocationEndpoint    string `json:"revocation_endpoint"`
+	JwksUri               string `json:"jwks_uri"`
+	Issuer                string `json:"issuer"`
+}
+
+type CodeTokenResponse struct {
+	AccessToken  string `json:"access_token"`
```

Comment:
can we use camelCase?

Reply:
In Golang, we need to uppercase the first character of `accessToken` to make it public.